README.md

```diff
@@ -1,10 +1,42 @@
-# React + Vite
+# PetSynth
+## AI Assisted Virtual Pet Generation
+`Group Participators: Conrad, Nico and Susannah`
```

Comment:
Is this a code block just to make it stand out? Should be Group Participators bit be separate?

Reply:
just to make it stand out - you have the visual example at the top